trucks.js

```diff
@@ -1,504 +1,684 @@
 		description: 'Gourmet, authentic falafel & shawarma',
 		website: 'http://falafelsalam.com',
 		Facebook: 'https://www.facebook.com/FalafelSalam',
-		Twitter: 'https://twitter.com/FalafelSalam'
+		Twitter: 'https://twitter.com/FalafelSalam',
+        read: false
 	},
 	{
-		name: 'Fez',
+		_id: 55,
+        name: 'Fez',
 		type: ['Mediterranean'],
 		schedule: ['Tuesday', 'Wednesday', 'Thursday', 'Friday'],
 		payment: ['Cash', 'Cards'],
 		description: 'Modern Middle eastern street food (and drink)!',
 		Facebook: 'https://www.facebook.com/FezOnWheels/info',
-		Twitter: 'https://twitter.com/fezonwheels'
+		Twitter: 'https://twitter.com/fezonwheels',
+        read: false
 	},
 	{
-		name: 'Flair Taco',
+		_id: 56,
+        name: 'Flair Taco',
 		type: ['Mexican'],
 		schedule: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
 		payment: ['Cash'],
 		description: 'Regular taco truck in the Fremont neighborhood - open 7 days a week and late night',
-		Facebook: 'https://www.facebook.com/pages/Flair-Taco/197832600252403'
+		Facebook: 'https://www.facebook.com/pages/Flair-Taco/197832600252403',
+        read: false
 	},
 	{
-		name: 'Frank\'s Franks',
+		_id: 57,
+        name: 'Frank\'s Franks',
 		type: ['Hot Dogs'],
 		schedule: ['Monday', 'Sunday'],
 		payment: ['Cash', 'Cards'],
 		description: 'Gourmet hot dogs, weiners, sausages, hot links and more, contact at 425.761.9447 or franksfranks3@gmail.com',
-		Facebook: 'https://www.facebook.com/franksfrankshotdogs'
+		Facebook: 'https://www.facebook.com/franksfrankshotdogs',
+        read: false
 	},
 	{
-		name: 'Fruit Chatter Box',
+		_id: 58,
+        name: 'Fruit Chatter Box',
 		type: ['Sweets'],
 		schedule: ['Sunday'],
 		payment: ['Cash', 'Cards'],
 		description: 'Fruit Chatter Box provides fresh, seasonal fresh fruit in cups or on a stick at local festivals and street fairs.',
 		Facebook: 'https://www.facebook.com/fruitchatterbox',
-		Twitter: 'https://twitter.com/fruitchatterbox'
+		Twitter: 'https://twitter.com/fruitchatterbox',
+        read: false
 	},
 	{
-		name: 'Full Tilt Ice Cream',
+		_id: 59,
+        name: 'Full Tilt Ice Cream',
 		type: ['Sweets'],
+        schedule: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
 		payment: ['Cash', 'Cards'],
 		description: 'Handmade ice creams!',
 		website: 'http://www.fulltilticecream.com',
 		Facebook: 'https://www.facebook.com/fulltilticecreams',
-		Twitter: 'https://twitter.com/fticecream'
+		Twitter: 'https://twitter.com/fticecream',
+        read: false
 	}
-];
\ No newline at end of file
+];
+
+var _ = require('underscore');
+  
+function filterByDay(day){
+//  var trucksFilter = _.filter(foodTrucks, function(day));  
+    return _.filter(foodTrucks, function(truck){
+          return _.contains(truck.schedule, day);
+    });
+}
+                  
+function getTrucks() {
+  return truckList;
```

Comment:
You're trying to return truckList, but truckList isn't defined. I assume you meant to say foodTrucks here.